benchmarks/allocation/alloc.cc

```diff
@@ -34,7 +35,10 @@ void __cheri_compartment("allocbench") run()
 		auto quota = heap_quota_remaining(MALLOC_CAPABILITY);
 		Debug::Invariant(quota == MALLOC_QUOTA, "Quota remaining {}, should be {}", quota, MALLOC_QUOTA);
 		Debug::log("Flushing quarantine");
-		heap_quarantine_empty();
+		Debug::Invariant(heap_quarantine_empty() >= 0,
+		                 "Unable to call heap_quarantine_empty");
```

Comment:
Hm.  I worry that "Unable to call" is potentially misleading... In many cases, the callees indeed cannot fail, so it's accurate, but in the case where they _can_ fail?

Reply:
Made the message clearer throughout